src/components/Navbar/NavBar.tsx

```diff
@@ -1,25 +1,82 @@
-import React from 'react';
+import React, { useState, useEffect, Dispatch } from 'react';
 import styles from './styles.module.scss';
 import { Link } from 'react-router-dom';
+import MenuIcon from '@material-ui/icons/Menu';
+import CloseIcon from '@material-ui/icons/Close';
 import sclLogo from '../../images/logo-with-text.png';
 
-interface Props {}
+interface Props {
+  menuOpen: boolean;
+  setMenuOpen: React.Dispatch<React.SetStateAction<boolean>>;
+}
 
 const NavBar = (props: Props) => {
-  return (
-    <div className={styles.navBarContainer}>
-      <img className={styles.image} src={sclLogo} alt="Logo" />
-      <div className={styles.navLinksContainer}>
-        <Link to="/" className={styles.navLink}>
+  const [hamburgerOpen, setHamburgerOpen] = useState(false);
+
+  const handleResize = () => {
+    if (window.innerWidth < 767) {
+      setHamburgerOpen(true);
+    } else {
+      setHamburgerOpen(false);
+    }
+  };
+
+  useEffect(() => {
+    if (window.innerWidth < 767) {
+      setHamburgerOpen(true);
+    } else {
+      setHamburgerOpen(false);
+    }
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  const showCompactMenu = () => {
+    return !props.menuOpen ? (
+      <MenuIcon onClick={(e) => props.setMenuOpen(true)} />
+    ) : (
+      <div className={styles.navLinksCompactContainer}>
+        <div className={styles.navBarContainer}>
+          <img className={styles.image} src={sclLogo} alt="Logo" />
+          <CloseIcon
+            onClick={(e) => props.setMenuOpen(false)}
+            className={styles.closeIcon}
+          />
+        </div>
+        <Link to="/" className={styles.navCompactLink}>
           HOME
         </Link>
-        <Link to="/sellers" className={styles.navLink}>
+        <Link to="/sellers" className={styles.navCompactLink}>
           MERCHANTS
         </Link>
-        <Link to="/about" className={styles.navLink}>
+        <Link to="/about" className={styles.navCompactLink}>
           ABOUT US
         </Link>
       </div>
+    );
+  };
+
+  return (
+    <div className={styles.navBarContainer}>
+      <img className={styles.image} src={sclLogo} alt="Logo" />
+      {hamburgerOpen ? (
+        showCompactMenu()
+      ) : (
+        <div className={styles.navLinksContainer}>
+          <Link to="/" className={styles.navLink}>
+            HOME
+          </Link>
+          <Link to="/sellers" className={styles.navLink}>
+            MERCHANTS
+          </Link>
+          <Link to="/about" className={styles.navLink}>
```

Comment:
I don't think we have this page implemented in Weebly so you can remove it here.

Reply:
the path for all of our merchants is currently /sellers right now. I could change the path, but i would rather do that in a new branch